src/work-allocation/containers/my-tasks/my-tasks.component.spec.ts

```diff
@@ -9,11 +9,14 @@ import {of} from 'rxjs';
 import {SessionStorageService} from 'src/app/services';
 
 import {WorkAllocationComponentsModule} from '../../components/work-allocation.components.module';
+import {WorkAllocationRelease2ComponentsModule} from '../../../work-allocation/release2/components/work-allocation.components.module';
```

Comment:
Imports should have spaces within the curly braces. Some in here do, most don't.